### DISCUSSION TITLE
Template ready to use

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1463] [Ideas / Feature Requests]
I think it's a good idea to develop such template like this [Template](https://panel.holoviz.org/getting_started/core_concepts.html) .we could add items to the side bar or to the top bar or the content page .  The template is ready for use .all we need to do is add elements to the template.  We could adjust the appearance of the template through some options

**Comment by falkoschindler** [comment]
Thanks for the proposal, @v479038280. This sounds very similar to #1461. Should we continue this discussion over there?

**Reply by v479038280** [reply]
>Can you give a specific example for such a template? What would some dummy code and the resulting layout look like?

They are the same idea,  https://panel.holoviz.org/getting_started/core_concepts.html.  may be  an example

**Reply by rodja** [reply]
Please repost your link on #1461 which was created a bit earlier than this one here. I'll close this duplication.
